circle.yml

```diff
@@ -0,0 +1,68 @@
+machine:
+  environment:
+    # We need to set this variable to let Anaconda take precedence
+    PATH: "/home/ubuntu/miniconda/envs/circleenv/bin:/home/ubuntu/miniconda/bin:$PATH"
+    MNE_DATA: "/home/ubuntu/mne_data"
```

Comment:
It looks for the env variable for the data path. See https://github.com/mne-tools/mne-python/blob/master/mne/datasets/utils.py#L101. We probably don't want to change that.
We discussed this with @agramfort and unless the MNE_DATA variable is set, the data is copied to mne-python/examples.